WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingLaunchStoreView.swift

```diff
@@ -73,13 +80,63 @@ struct StoreOnboardingLaunchStoreView: View {
             }
         }
         .navigationTitle(Localization.title)
+        .task {
+            await viewModel.checkEligibilityToPublishStore()
+        }
+    }
+
+    var freeTrialBanner: some View {
```

Comment:
Nit: this should be private.

Reply:
Done in f54e69816c981c2b6d9e72c0b4acef776fac3bd0